### ISSUE TITLE
General suggestions

### ISSUE THREAD

**Issue by kingrollo** [jtreml/leaflet.measure#4]
I really like this plugin - it's much cleaner than the other measure tool that's floating around. You should submit it to the full Leaflet plugins page.

There are two things I'd change (have changed actually in my implementation). Firstly, it would be nice if it was more easily configurable for different units - even if it just meant passing the calculation as a formula to the library when loaded. Secondly, the icon isn't too intuitive - I've shared the one I'm using below...

![ruler](https://f.cloud.github.com/assets/3293453/760576/3036d4ec-e7b1-11e2-9b7e-e8b570616a1b.png)

Thanks!

**Comment by kartenkarsten** [comment]
I totaly agree adding to the Leaflet Plugin list would be really practical.

**Comment by justinlewis** [comment]
I agree that there should be an option for setting units. You can add "options" to the extended Polyline class of leaflet.measurecontrol.js but this is just a hack. 

```
L.Polyline.Measure = L.Draw.Polyline.extend({
    options: {
        metric: false
  },
 ....
```

It would be nice to set these options when adding the control to the map (i.e. L.Control.measureControl({metric:false}).addTo(map)).

I'll try to come back to this when I have more time.

**Comment by jonathandarrer** [comment]
Great plugin, I'm using it at work. I've added map events to start/stop measuring in my fork, https://github.com/jonathandarrer/leaflet.measure which my company is now using. If you want to pull this change into your repository, then feel free to. What do you think?

``` js
_startMeasuring: function() {
  ...
  this._map.fire('measure:measurestart', { layerType: this.type });
  ...
}

_stopMeasuring: function() {
  ...
  this._map.fire('measure:measurestop', { layerType: this.type });
  ...
}
```

**Comment by activemkaye** [comment]
@kingrollo @justinlewis 
I know this code hasn't been updated in years, but it's exactly what I'm looking for. I just don't know what the unit "nm" is? I tried conversions to feet for miles, nano meters, and nautical miles. None of these makes sense. The current API of the leaflet distanceTo method returns meters. Can anyone chime in here?
